### DISCUSSION TITLE
global threat event  - Purpose

### DISCUSSION THREAD

**Discussion by rsrinivasanhome** [simonec73/threatsmanager#15] [Q&A] [answered]
I need an ad hoc threat event to be added to my threat model.  It was a threat identified based on brainstorming with the team.

I followed the below procedure to create a threat event

- Create a new Threat type
- Create a new Global Threat event for the threat type

Is this the correct purpose of global threat event ?

**Comment by simonec73** [comment, marked as answer]
Thank you for asking! If you identify a new threat, you can simply select the object which must receive it (entity, flow or threat model) and then add a Threat event. This will create the associated threat type automatically.
The Global Threat Event us simply a Threat event which is associated to the threat model, thus is global.
If it answers your question, please mark it as answer.

**Reply by rsrinivasanhome** [reply]
In short a global threat event is not related to any  entity or flow. (It is at the threat model level)

**Reply by simonec73** [reply]
Correct
